src/index.js

```diff
@@ -13,7 +14,7 @@ app.use(express.json());
 app.use(express.urlencoded({extended: true}));
 
 // Routes
-// app.use("/example", exampleRouter)
+app.use("/user", userRouter);
```

Comment:
make user plural.
Rest APIs should be plural for the groupings and then if we need a specific user we would make the path /users/:id where the id would identify a specific user within the users grouping

Reply:
You're right, this should be plural